packages/gateway/src/api/Gateway.ts

```diff
@@ -0,0 +1,15 @@
+// import { Api } from '@scalecube/scalecube-microservice';
+
+export default interface Gateway {
+  /**
```

Comment:
remoteCall require router in-order to pick the "right" service
------
router: Router